Samples/MedicationsTracker/MedicationsTrackerTests/EditMedicationViewControllerTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  EditMedicationViewControllerTests.swift
+//  MedicationsTracker
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import MedicationsTracker
+
+class EditMedicationViewControllerTests: XCTestCase
+{
+    var editViewController : EditMedicationViewController?
+    var medField: MedicationTextField?
+    var pickerField: PickerTextField?
+    var nameField: MedicationTextField?
+    var medThing: MHVThing? = nil
+    
+    override func setUp()
+    {
+        super.setUp()
+        
+        medThing = MHVMedication.newThing()
+        let med = medThing?.medication()
+        med?.strength = MHVApproxMeasurement.fromValue(500, unitsText: "Milligrams (mg)",
+                                       unitsCode: "Milligrams (mg)", unitsVocab: "medication-strength-unit")
+        med?.dose = MHVApproxMeasurement.fromDisplayText("A single tablet")
+        editViewController = EditMedicationViewController.init(nibName: "EditMedicationViewController", bundle: nil,
+                                                              medication: medThing!,
+                                                              builder: MedicationBuilder())
+        med?.name = MHVCodableValue.fromText("Advil")
+        editViewController?.nameField = AutocompleteTextField()
+        editViewController?.strengthAmountField = MedicationTextField()
+        editViewController?.strengthUnitField = PickerTextField()
+        editViewController?.doseAmountField = MedicationTextField()
+        editViewController?.doseUnitField = PickerTextField()
```

Comment:
Try doing editViewController?.view.setNeedsLayout()  That should make it load the XIB file & setup the text fields, so you don't have to create in code